apps/codesearch/deployment.yaml

```
@@ -25,7 +25,7 @@ spec:
       - name: codesearch-fetch
         # Placeholder image until we can add image under control of k8s.io
         # Issue to track this work: https://github.com/kubernetes/k8s.io/issues/2182
-        image: jdagostino2/codesearch-fetch:0.1.7
+        image: gcr.io/k8s-staging-infra-tools/cs-fetch-repos
```

```suggestion
        image: gcr.io/k8s-staging-infra-tools/cs-fetch-repos:v20220317-c16ef4289
```

Let's set the tag used for this image.